src/git-version.cr

```diff
@@ -14,10 +14,25 @@ module GitVersion
   MINOR_BUMP_COMMENT = "feature:"
 
   class Git
-    def initialize(@dev_branch : String, @release_branch : String = "master" , @folder = FileUtils.pwd)
+    def initialize(@dev_branch : String, @release_branch : String = "master", @patch_cmd : (String | Nil) = nil, @folder = FileUtils.pwd)
       #
     end
 
+    private def attemptExec(cmd, ver)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name should be underscore-cased: attempt_exec, not attemptExec](https://app.codacy.com/gh/codacy/git-version/pullRequest?prid=4909832)